Templates/JenkinsPipeline/README.md

```diff
@@ -0,0 +1,132 @@
+
+**Note: Please work with your pipeline specialist to review the below section.**
+
+The `Jenkinsfile` can be placed into the root folder of your Git repository and will automatically provide pipelines for the specified triggers defined in the Jenkins Multibranch Pipeline job configuration.
+
+Please review the definitions thoroughly with your Jenkins administrator. Ideally, the Jenkinsfile is converted into a [Jenkins Shared Library](https://www.jenkins.io/doc/book/pipeline/shared-libraries/) to provide stronger, central control.
+
+Step-by-step instructions to define and configure a Jenkins Multibranch Pipeline is documented at: https://www.jenkins.io/doc/book/pipeline/multibranch/
+
+
+## Pipeline usage
+
+The pipeline template implements the common clone, build, package and deploy steps to process various configurations according to the defined conventions. It leverages the a declarative checkout of the application's Git repository.
+
+It is supporting the following workflows: 
+
+* automated [build pipelines for feature branches](https://ibm.github.io/z-devops-acceleration-program/docs/branching-model-supporting-pipeline#pipeline-build-of-feature-branches) with a clone and build stage
+* [basic pipeline](https://ibm.github.io/z-devops-acceleration-program/docs/branching-model-supporting-pipeline#the-basic-build-pipeline-for-main-epic-and-release-branches) triggered when feature branches are merged into the branch `main`
+* [release pipeline](https://ibm.github.io/z-devops-acceleration-program/docs/branching-model-supporting-pipeline#the-release-pipeline-with-build-packaging-and-deploy-stages) to build, package and deploy a release via UCD
+
+The `Pipeline setup` stage within the [template](Jenkinsfile#L165) configures the subsequent stages, including flags used as conditions when stages and jobs are executed. Please review this stage.
+
+This template is implementing the recommended [Git branching model for mainframe development](https://ibm.github.io/z-devops-acceleration-program/docs/git-branching-model-for-mainframe-dev). If you haven't done yet, we recommend to get familiar with the concepts described in this documentation.
+
+### Pipeline variables
+
+In a default setup, the feature and basic pipelines are triggered for each new commit as soon as the Multibranch integration detects the changes.
+
+The pipeline allows to override the values of the below variables when manually requesting the pipeline. This is especially useful when the application team want to create a release candidate package for higher test environments and production.
+
+Parameter | Description
+--- | ---
+pipelineType     | Pipeline type - either build, release or preview. (Default: build)
+verbose          | Boolean flag to control logging of build framework. (Default: false)
+<!-- releaseType      | Release type - major, minor, patch as input to compute the release version and to set the release candidate and release git tags. (Default: patch) //-->
```

Comment:
Should the `releaseType` row be uncommented?

Reply:
The current implementation is not computing the next planned release. I think that would be a good enhancement to include into this PR.